web/files/files_test.go

```diff
@@ -764,6 +764,59 @@ func TestUploadWithSourceAccount(t *testing.T) {
 	assert.Equal(t, identifier, fcm["sourceAccountIdentifier"])
 }
 
+func TestCopyFile(t *testing.T) {
+	_, foo := createDir(t, "/files/?Name=foo&Type=directory")
+	fooID := foo["data"].(map[string]interface{})["id"].(string)
+
+	fileName := "bar"
+	fileContent := "file content"
+
+	// 1. Upload file and get its id
+	res, obj := upload(t, "/files/"+fooID+"?Type=file&Name="+fileName, "text/plain", fileContent, "")
+	assert.Equal(t, 201, res.StatusCode)
+	data := obj["data"].(map[string]interface{})
+	fileID = data["id"].(string)
+	fileAttributes := data["attributes"].(map[string]interface{})
+
+	var body map[string]interface{}
+
+	// 2. Send file copy request
+	res, _ = httpPost(ts.URL+"/files/"+fileID+"/copy", "")
+	assert.Equal(t, 201, res.StatusCode)
+	assert.NoError(t, json.NewDecoder(res.Body).Decode(&body))
```

Comment:
You can use `require.NoError` when you want the test to stop in case of error. It is not used often as `require` has been added several years after `assert` and I also discovered it long time after it was added. But it can help to avoid lots of error when something got wrong in the first few steps.

Reply:
I've actually been using it a lot in the other tests. I'll update this one !